.github/workflows/build-stable-apk.yml

```
@@ -0,0 +1,49 @@
+          sed -n '/signingConfigs {/,/    }/p' app/build.gradle.kts > signing_block.txt
+          # Remove original signingConfigs block
+          sed -i '/signingConfigs {/,/    }/d' app/build.gradle.kts
+          # Insert signingConfigs block before buildTypes
+          sed -i '/buildTypes {/e cat signing_block.txt' app/build.gradle.kts
+          # Link release build to debug signing
+          sed -i '/release {/a \            signingConfig = signingConfigs.getByName("debug")' app/build.gradle.kts
+
+      - name: Build Github Release APK
+        run: ./gradlew assembleGithubRelease
+
+      - uses: "marvinpinto/action-automatic-releases@latest"
```

actionlint (1.7.12)

[error] the runner of "marvinpinto/action-automatic-releases@latest" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)